CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/UserCommentUpdaterCli.java

```diff
@@ -0,0 +1,30 @@
+package org.eupathdb.sitesearch.data.comments;
+
+public class UserCommentUpdaterCli extends CommentUpdaterCli {
+  
+  public static void main(String[] args) throws Exception {
+    UserCommentUpdaterCli cli = new UserCommentUpdaterCli();
+    cli.init(new UserCommentSolrDocumentFields(), new UserCommentUpdaterSql());
```

Comment:
Why not pass these to the abstract class constructor rather than init()?  Looks like the class is in an invalid state before init() is called.